### DISCUSSION TITLE
Negative pressure in a fully enclosed compartment

### DISCUSSION THREAD

**Discussion by MNJOLE** [firemodels/fds#13001] [Q&A] [answered]
Hello, I would like to create a situation where the cooler works in a perfectly sealed compartment.
I haven't placed the burner yet, but I plan to add it later. (I am planning to apply the pressure zone when implementing it like this.)
However, when the environment is implemented, it is confirmed that the pressure is not maintained, but that the pressure drops abnormally.
I didn't apply any leak or install vent for pressure relief.
The HVAC-based cooler simply operates.
My prediction was that the pressure would decrease as the temperature drops, but the results were hard to understand.(negative pressure)
And the additional problem is that if no input is made to the MISC line, I understand that it is designated as default, 20 degrees Celsius, 101 kPa, but the initial pressure of the compartment being interpreted is 0 kPa.
Can you advise me on what to do to alleviate the problem?

**Comment by drjfloyd** [comment, marked as answer]
As stated in the FDS Technical Reference Guide, FDS uses the Ideal Gas Law as the Equation of State:

PV = n R T

If you have a sealed compartment then V (volume) is constant, the gas constant R is constant, and n (moles of gas) is constant. What happens to P (pressure) if T (temperature) decreases?

When looking at pressure outputs make sure you understand what quantity you are looking at. Some pressure outputs are total pressure (i.e. 101 kPa at t=0) and some outputs are the perturbation pressure (0 kPa at t=0).

**Reply by MNJOLE** [reply]
Yes, I thought the pressure would decrease as well, but the calculations initially resulted in a pressure of zero, and then showed extremely low negative pressure values, which seemed problematic. However, as you pointed out, it seems the issue was with the measurement being taken as perturbation pressure. Thank you for your quick and clear response!
